.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2024-03-26 - Prevent Zombie Processes via kill_on_drop
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)